entrypoint/secretsencrypt/encryption_suite_test.go

```diff
@@ -51,23 +52,14 @@ func TestSecretsEncryptionSuite(t *testing.T) {
 	RunSpecs(t, "Secrets Encryption Test Suite")
 }
 
-var _ = BeforeSuite(func() {
-	if cluster.Config.Product == "k3s" {
-		Expect(os.Getenv("server_flags")).To(ContainSubstring("secrets-encryption:"),
-			"ERROR: Add secrets-encryption:true to server_flags for this test")
+func validateSecretsEncryptFlag() {
+	if cfg.Product == "k3s" {
+		if !strings.Contains(os.Getenv("server_flags"), "secrets-encryption:") {
+			shared.LogLevel("error", "Add secrets-encryption:true to server_flags for this test")
```

Comment:
`Add secrets-encryption: true`

Reply:
Some day, we may want to test 'secrets-encryption: false' for an issue or something, say. The odds of that happening are really minimal. But if we put 'true' in, then the test wont run at all, and we will miss out an opportunity to reuse some existing automation. Thats why i did not add the true keyword there. Also, if someone has an extra space between 'secrets-encryption:  true'  would it fail then - and we check for contains 1 space - it would fail again - but falsely? We shouldnt limit this check to the exact value.

Reply:
I meant add an space on line 58 between `secrets-encryption:` and `true` so that it says `Add secrets-encryption: true to server_flags for this test`